test/unit/util_color.js

```diff
@@ -0,0 +1,62 @@
+var test = require('tap').test;
+var color = require('../../src/util/color');
+
+test('decimalToHex', function (t) {
+    t.equal(color.decimalToHex(0), '#000000');
+    t.equal(color.decimalToHex(1), '#000001');
+    t.equal(color.decimalToHex(16777215), '#ffffff');
+    t.equal(color.decimalToHex(-16777215), '#000001');                  // @todo
```

Comment:
Hmmm... I think all our decimal colors are represented by positive integers? So while not technically correct, I'd venture this may be reasonable.